flink-connector-bigquery/src/main/java/com/google/cloud/flink/bigquery/sink/writer/BigQueryDefaultWriter.java

```diff
@@ -40,20 +47,67 @@
  *
```

Comment:
same comment as above. for example, when write is called, data gets added to the buffer and is asynchronously appended to BQ row when flushed, etc.

Reply:
i think what oyu have is sufficient (in conjuncture w/ the javadoc on the basewriter). you can resolve this!